db/migrate/20170519181110_add_description_to_collections.rb

```diff
@@ -0,0 +1,6 @@
+class AddDescriptionToCollections < ActiveRecord::Migration
+  def change
+    add_column :collections, :description, :text, default: ""
```

Comment:
Is 60k "high volume"? I knew about backfilling and was just avoiding a multiple-deploy resolution. I guess any table lock on collections is bad and I'll do the above, but what's the cutoff for this?